Sources/ExyteChat/Views/MessageView/MessageMenu/MessageMenu+Action.swift

```diff
@@ -65,11 +65,30 @@ public enum DefaultMessageMenuAction: MessageMenuAction, Sendable {
         .copy, .reply, .edit(saveClosure: {_ in})
     ]
     
-    static public func menuItems(for message: Message) -> [DefaultMessageMenuAction] {
+    static public func menuItems(
+        for message: Message, availableMessageMenuItems: [AvailableMesssageMenuType]
+    ) -> [DefaultMessageMenuAction] {
+
```

Comment:
This is a fairly simple change to allow the user to set which menu options are shown